specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,437 @@ with interoperability issues.
 New APIs introduced by DWriteCore include:
 
  - The exported function is renamed `DWriteCoreCreateFactory` to avoid a naming collision with the
-   `DWriteCreateFactory` function exported by the Windows DLL (DWrite.dll).
- - A new DWRITE_FACTORY_TYPE enumeration for creating an isolated factory object
+   `DWriteCreateFactory` function exported by the system version of DirectWrite (DWrite.dll)
+ - A new `DWRITE_FACTORY_TYPE` enumerator for creating an isolated factory object
```

Comment:
The constants in an enumeration are called enumerators.

See also:
- https://docs.microsoft.com/en-us/cpp/cpp/enumerations-cpp?view=msvc-170
- https://en.cppreference.com/w/cpp/language/enum

Reply:
RECOMMEND: The name of the thing is an enumeration, which is a set of named/grouped constants.

Reply:
Is there any basis for `enumerant`? Making up words that aren't used in the general community is unhelpful for a public specification. Perhaps this can be refactored out as such:

```suggestion
 - Adds `DWRITE_FACTORY_TYPE` to create an isolated factory object
```

Reply:
[The Windows Runtime calls them "values"](https://github.com/microsoft/xlang/blob/master/docs/xplatwinrt/XSPEC02%20-%20Metadata%20Representation.md):

> After the enum value definition comes a field definition for each of the values in the enumeration

That seems like a word whose meaning is clear to all readers, despite perhaps not matching what C++ calls it (because the C++ name is easily confused with C# enumerators).

>A new `DWRITE_FACTORY_TYPE` value which specifies...

Reply:
@oldnewthing <shakes hand> I like it; agreed!

Reply:
Sounds good. I'm going with:
A new `DWRITE_FACTORY_TYPE` value which specifies an isolated factory object